scripts/generateRecipeTable.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+echo "# Recipes"
+echo
+echo "| Recipe | Description | Node | Browser |"
+echo "| - | - | - | - |"
+for d in recipes/*; do
+  base=$(basename $d)
+  description=$(sed '3q;d' $d/README.md)
```

Comment:
We'll need to add instructions that clarify that the Recipe description must always be on the 3rd line in the file and cannot span across multiple lines.